<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherMenuWidget</class>
 <widget class="QWidget" name="teacherMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>119</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>排序窗口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;教师登录界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
